packages/strapi-admin/admin/src/containers/LeftMenu/utils/tests/getCtOrStLinks.test.js

```diff
@@ -0,0 +1,128 @@
+import { request, hasPermissions } from 'strapi-helper-plugin';
+import getCtOrStLinks from '../getCtOrStLinks';
+
+jest.mock('strapi-helper-plugin');
+
+describe('checkPermissions', () => {
+  beforeEach(() => {
+    hasPermissions.mockImplementation(() => Promise.resolve(true));
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('creates an array of boolean corresponding to the permission state', async () => {
+    const userPermissions = [
+      {
+        id: 458,
+        action: 'plugins::i18n.locale.read',
+        subject: null,
+        properties: {},
+        conditions: [],
+      },
+      {
+        id: 460,
+        action: 'plugins::content-manager.explorer.read',
+        subject: 'application::article.article',
+        properties: {
+          fields: ['Name'],
+          locales: ['en'],
+        },
+        conditions: [],
+      },
+      {
+        id: 461,
+        action: 'plugins::content-manager.explorer.read',
+        subject: 'application::article.article',
+        properties: {
+          fields: ['Name'],
+          locales: ['fr-FR'],
+        },
+        conditions: [],
+      },
+    ];
+
+    const data = [
+      {
+        uid: 'application::address.address',
+        isDisplayed: true,
+        apiID: 'address',
+        kind: 'collectionType',
+        info: {
+          label: 'address',
+        },
+      },
+      {
+        uid: 'application::article.article',
+        isDisplayed: true,
+        apiID: 'article',
+        kind: 'collectionType',
+        info: {
+          label: 'article',
+        },
+        pluginOptions: {
+          i18n: {
+            localized: true,
+          },
+        },
+      },
+    ];
+
+    request.mockImplementation(() => Promise.resolve({ data }));
+
+    const expected = {
+      authorizedCtLinks: [
+        {
+          destination: '/plugins/content-manager/collectionType/application::address.address',
+          icon: 'circle',
+          isDisplayed: false,
```

Comment:
Since we are not using the isDisplayed key can you remove it?

Reply:
This one comes from the backend 😓